tests/baselines/reference/publicGetterProtectedSetterFromThisParameter.errors.txt

```diff
@@ -0,0 +1,45 @@
+tests/cases/compiler/publicGetterProtectedSetterFromThisParameter.ts(33,7): error TS2446: Property 'q' is protected and only accessible through an instance of class 'A'. This is an instance of class 'B'.
+tests/cases/compiler/publicGetterProtectedSetterFromThisParameter.ts(34,7): error TS2446: Property 'u' is protected and only accessible through an instance of class 'A'. This is an instance of class 'B'.
+
+
+==== tests/cases/compiler/publicGetterProtectedSetterFromThisParameter.ts (2 errors) ====
+    class A {
+      get x() { return 0; }
+      protected set x(v: number) { }
+    
+      public get y() { return 0; }
+      protected set y(v: number) { }
+    }
+    
+    class B {
+      get q() { return 0; }
+      protected set q(v: number) { }
+    
+      protected get u() { return 0; }
+      protected set u(v: number) { }
+    
+      foo(this: A, a: A, b: B) {

```

Comment:
highly recommend switching from `foo/bar/baz` to `donkey/diddy/cranky`

Reply:
bananas